demo-shell/src/app/components/search/search-result.component.html

```diff
@@ -1,19 +1,24 @@
-<adf-search [searchTerm]="searchedWord"
-            [maxResults]="maxItems"
-            [skipResults]="skipCount"
-            (resultLoaded)="onSearchResultLoaded($event)"
-            #search>
-</adf-search>
+<div class="adf-search-results__facets">
```

Comment:
if we add the new custom search here we don't test anymore the adf-search component in the demo shell could it be better have two separate test

Reply:
Do we really need two search implementations? Given that first one has no design and requirements? I though we will have only latest one

Reply:
We still have auto-completion and templating for the search input, the only thing that is replaced is "search results" page. Correct me if I'm wrong, but this is the "real" search results as expected from us, from what I understand, and previous implementation can be removed.

Reply:
Denys is correct here, as long as we still have the "live search" (with autocomplete) with the customisable template this is good enough. The full search results page should be the new one with everything.

Reply:
If we remove it is a BC can notbe removed

Reply:
We still need to test it so is implementation need to be in the demo shell otherwise the Way team can not keep to check those functionality

Reply:
The first implementation had requirements and I think this one should keep to work with it I will set up a meeting monday

Reply:
Ok, I will restore old demo and have faceted one as a separate page with separate input

Reply:
@eromano @ohej I will have two pages to have both demos up and running in the demo shell

Reply:
@DenisVuyka I don't think honestly is just material of two pages . I think is more mater of make the new search a lint extension of the previous code and I think there is room to do it

Reply:
Guys, can we have an architecture design for the entire epic before I spend 18 days fleshing out the container? It's it's not something ADF wants, then I need to know what exactly ADF wants